lib/view/formulier/InvoerVelden.class.php

```diff
@@ -838,7 +838,8 @@ public function getHtml() {
 		$html = '<input name="' . $this->name . '_show" value="' . $show_value . '" origvalue="' . $this->show_value . '"' . $this->getInputAttribute(array('type', 'id', 'class', 'disabled', 'readonly', 'maxlength', 'placeholder', 'autocomplete')) . ' />';
 
 		// actual values
-		$class = $this->model->orm;
+        $model = $this->model;
```

Comment:
spaties > tabs :grin: Iedere goede IDE kan hier toch mee overweg.

Daar ben ik nog mee aan het kloten, hij werkte niet, want `orm` is een const, dus static, dus `::orm`.